generation_config.yaml

```diff
@@ -944,10 +946,11 @@ libraries:
   - proto_path: google/cloud/functions/v2alpha
   - proto_path: google/cloud/functions/v2beta
 
-- api_shortname: gke-backup
+- api_shortname: gkebackup
```

Comment:
`api_shortname` is used to generate the folder name, to keep the folder name the same, please add the following
```
library_name: gke-backup
```